site/docs/skylark/aspects.md

```diff
@@ -75,38 +75,72 @@ example. An aspect implementation function is then invoked on all nodes in
 the shadow graph similar to how rule implementations are invoked on the nodes
 of the original graph.
 
-## Defining aspects
+## Simple example
 
-Aspect definitions are similar to rule definitions, and defined using
-the [`aspect`](lib/globals.html#aspect) function. Let's take a look at
-the example:
+This example demonstrates how to recursively print the source files for a 
+rule and all of its  dependencies that have a `deps` attribute. It shows 
+how to define an empty provider, an aspect implementation and an aspect 
```

Comment:
"implementation," - we use the serial comma at Google

Reply:
Fixed.